MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```
@@ -18,15 +18,22 @@ final class FeedViewModel {
         Localized.Feed.title
 	}
 
+    var errorMessage: String = Localized.Feed.loadError
+
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
-	
+    var onErrorMessage: Observer<String>?
```

`onErrorMessage: Observer<String> ` could be `onErrorStateChange: Observer<String?>`, so the ViewModel decides when to show/hide errors (instead of the view controller). It's often easier to manage state when all changes are centralized.

For example, in the `loadFeed()` method, the ViewModel can call  `onErrorStateChange(.none)` to hide the error before loading the feed.